pages/common/from.md

```diff
@@ -0,0 +1,19 @@
+# from
+
+> Prints mail header lines from your mailbox.
+
```

Comment:
Link to more information would be nice.

Reply:
Hmm...I can't find anything about it.  I guess it's too generic :)

It's built-in, though, and you can't install/uninstall it.

Reply:
From the man page:
> The from command appeared in 3.0BSD.
>
> December 30, 1993

Reply:
@bl-ue we're talking about `from` from GNU Mailutils right?
Check - https://mailutils.org/manual/html_chapter/Programs.html#frm-and-from

On a side note, online documentations are more update to date so having a reference in `tldr-pages` is recommended.

Reply:
@sahilister it does look like it, good job for finding it! (except `frm` is not available on ubuntu or macOS)

Reply:
done @ feb34d4e1d41814051802a1ca470b50b4d78a04a